### ISSUE TITLE
Unicode characters small o and small p different widths

### ISSUE THREAD

**Issue by michaellindon** [JuliaLang/julia#13381]
I have a function which is causing me problems when accepting unicode characters 

```julia
julia> function foo(yₒ,yₚ)
ERROR: syntax: invalid character "ₚ"
```

I'm inputting these into the repl with `\_o<tab>` and `\_p<tab>`
Strangely it works with little 'o' but not with little 'p'.

```julia
julia> charwidth('ₒ')
1
julia> charwidth('ₚ')
0
```

why should these be different?

**Comment by yuyichao** [comment]
What julia version are you using? It works here on 0.5-dev. IIRC we have upgraded utf8proc version (twice?) in the 0.4 cycle

**Comment by jiahao** [comment]
You must be still be using Julia 0.3. Subscript-p ([U+209A](http://www.fileformat.info/info/unicode/char/209a/index.htm)) was introduced only in Unicode 6.0. Support for Unicode 8.0 and our fixes for charwidth are now in the 0.4 release candidate.

**Comment by michaellindon** [comment]
I was using 0.3, I just downloaded 0.4 and this resolved the issue, thank you.
